src/instruments/src/SD/Pages/Apu/Apu.tsx

```diff
@@ -0,0 +1,521 @@
+                                radius={50}
+                                startAngle={240}
+                                multiplierOuter={1.2}
+                                endAngle={60}
+                                className="NoFill AmberHeavy"
+                                outer
+                            />
+                            {apuN.isNormalOperation()
+                        && (
+                            <GaugeMarkerComponent
+                                x={0}
+                                y={50}
+                                min={0}
+                                radius={50}
+                                max={120}
+                                startAngle={240}
+                                endAngle={60}
+                                value={Number.parseFloat(apuN.value.toFixed())}
+                                className={`Line ${apuNIndicationColor}`}
+                                indicator
+                            />
+                        )}
+                        </GaugeComponent>
+                    </GaugeComponent>
+                </SvgGroup>
+
+                <SvgGroup x={80} y={13}>
+                    <text x={0} y={0} className="White Center FontNormal">N</text>
+                    <text x={0} y={30} className="Cyan Center FontNormal">%</text>
+                </SvgGroup>
+
+                <text
+                    x={10}
+                    y={80}
+                    className={`FontLarger Left ${apuN.isNormalOperation() ? apuNIndicationColor : 'Amber'}`}
+                >
+                    {apuN.isNormalOperation() ? apuN.value.toFixed() : 'XX'}
+                </text>
+            </SvgGroup>
+        </>
+    );
+};
+
+const EgtGauge = ({ x, y } : ComponentPositionProps) => {
+    const apuEgt = useArinc429Var('L:A32NX_APU_EGT', 100);
+    const displayedEgtValue = Math.round(apuEgt.value / 5) * 5; // APU Exhaust Gas Temperature is shown in steps of five.
+
+    const apuEgtCaution = useArinc429Var('L:A32NX_APU_EGT_CAUTION', 500);
+    const apuEgtWarning = useArinc429Var('L:A32NX_APU_EGT_WARNING', 500);
+
+    const [apuEgtIndicationColor, setApuEgtIndicationColor] = useState('Green');
+
+    const redLineShown = apuEgtCaution.isNormalOperation() && apuEgtWarning.isNormalOperation();
+
+    useEffect(() => {
+        if (apuEgt.value <= apuEgtCaution.value) {
+            setApuEgtIndicationColor('Green');
+        }
+        if (apuEgt.value > apuEgtCaution.value && apuEgt.value <= apuEgtWarning.value) {
+            setApuEgtIndicationColor('Amber');
+        }
+        if (apuEgt.value > apuEgtWarning.value) {
+            setApuEgtIndicationColor('Red');
+        }
+        // TODO FIXME: ecbAdvisoryDetected must be defined in a system later on.
+        // if (apuEgt.value <= apuEgtCaution.value && apuAvail && ecbAdvisoryDetected) {
+        //     setApuEgtIndicationColor('Pulse');
+        // }
```

Comment:
**thought**: TODO or FIXME?
**suggestion**: Avoid checking in commented out code.

Reply:
What do you think would be a good memo to put here so that we know to come back and implement this specific piece of logic in the future?